css/form.css~

```diff
@@ -0,0 +1,20 @@
+body.bg
```

Comment:
Remove this file. Its backup file created by Linux
Please make sure before commit, you cross check what you committing.